examples/equilibria/double_null_ST.py

```diff
@@ -315,7 +315,7 @@ def init_equilibrium(grid, coilset, constraint_set):
         li=None,
     )
     constraint_set(eq)
-    optimiser = UnconstrainedTikhonovCurrentCOP(eq, constraint_set, gamma=1e-7)
+    optimiser = UnconstrainedTikhonovCurrentGradientCOP(eq, constraint_set, gamma=1e-7)
```

Comment:
Missing `coilset` argument in `UnconstrainedTikhonovCurrentGradientCOP` - `double_null_ST.py` is being removed anyway in #1027, so this is not a major issue